package.json

```diff
@@ -59,6 +59,7 @@
     "fsevents": "~2.1.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.3",
```

Comment:
i couldn't get the dynamic to work without adding a new plugin that uses babel, let me know if you think that there is a better way.

Reply:
See my other comment. I would have time to take a deeper look tomorrow